### PR TITLE
VACMS-1436: Rename Decision Reviews menu.

### PR DESCRIPTION
**Pull request by kevwalsh**
## Description

See #1436

## Testing done
None yet. 

## QA steps

- [x] On PR environment created by this PR, publish all /decision-reviews benefits detail pages and then trigger a content build. 
- [x] decision-reviews/board-appeal/ should show a left nav

### CODE REVIEW

File: config/sync/system.menu.decision-reviews-benefits-h.yml [hunk 1]
```diff
@@ -9,6 +9,6 @@ third_party_settings:
     access_scheme:
       73: '73'
 id: decision-reviews-benefits-h
-label: 'Decision reviews benefits hub'
+label: 'Decision reviews'
```
Comment (ncksllvn) [inline config/sync/system.menu.decision-reviews-benefits-h.yml:12]: The `benefits hub` aligns with the naming conventions of the other hub sidebars, but the `fieldAdministration` property on the Decision Reviews pages is only `Decision reviews`. So updating that `fieldAdministration` on all the pages would be more consistent with the rest of the pages but probably more work

Comment (kevwalsh) [reply config/sync/system.menu.decision-reviews-benefits-h.yml:12]: oh, are we talking about _right_ rail? fieldAdministration matters for right rail on the hub landing page (which it shouldn't, but it does for now), but it really should never matter for the _left_ nav on benefits detail landing pages. i think a few of us should get on the phone to understand what problem we are trying to solve here.

Comment (kevwalsh) [reply config/sync/system.menu.decision-reviews-benefits-h.yml:12]: PR environment for review: http://pr1437.web.ci.cms.va.gov/decision-reviews/higher-level-review/

**Comment by swirtSJW** [comment]
image

**Pull request merged**